translate/storage/jsonl10n.py

```diff
@@ -322,43 +255,22 @@ class WebExtensionJsonFile(JsonFile):
     def _extract_translatables(self, data, stop=None, prev="", name_node=None,
                                name_last_node=None, last_node=None):
         for item in data:
-            yield (item, item, data[item], item)
-
-    def serialize_units(self):
-        units = OrderedDict()
-        for unit in self.unit_iter():
-            units[unit.getid()] = unit._node
-        return units
+            value = data[item]
```

Comment:
Is it not possible to do:

```python
        for item, value in six.iteritems(data):
```

instead of:

```python
        for item in data:
            value = data[item]
```